azure-functions/number-generator-function/activity-get-number-from-generator/index.js

```diff
@@ -34,24 +33,22 @@ const getNumberFromGenerator = async (context) => {
       throw new Error(JSON.stringify(`Invalid entityType: ${entityType}`));
   }
 
-  // Set the maximum number of tries in case acbsCheck is unavailable and it gets stuck in an infinite loop
+  /**
+   * Maximum tries mitigates infinite loop execution
+   */
   while (!numberIsAvailable && loopCount < MAX_NUMBER_OF_TRIES) {
-    console.info('Azure functions - getNumberFromGenerator - trying. Count ', loopCount);
-
-    numberFromGenerator = await numberGeneratorController.callNumberGenerator(numberType);
-    console.info('Azure functions - numberFromGenerator ', numberFromGenerator);
+    console.info(`⚡️ Execution #${loopCount}`);
```

Comment:
Should we add a description about what it's executing? I e. Executing number generator